### PR TITLE
Bug/001 fix all issues

### PR DESCRIPTION
**Pull request by elanghel**

### CODE REVIEW

File: ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift [hunk 1]
```diff
@@ -86,10 +79,12 @@ class AddTaskViewController: UIViewController {
         
         return view
     }
+    
 }
 
 extension AddTaskViewController: CategoryButtonDelegate {
     func didClickButton(category: TaskCategories) {
+        print("hello")
```
Comment (vtoloconnicov) [inline ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift:87]: remove prints. they shouldn't be in main branch

File: ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift [hunk 2]
```diff
@@ -42,34 +32,32 @@ class HomeTableViewCell: UITableViewCell {
...
-        }
+    func isSelected() {
+        isChecked = !isChecked
+        checkImage.tintColor = isChecked ? UIColor(named: "Check Color") : UIColor(named: "Background Color")
+        taskView.backgroundColor = isChecked ? UIColor(named: "Selected Task Color") : UIColor(named: "Task Color")
     }
     
     
-    func setup(title: String, importance: Bool) {
+    func setup(title: String, category: String, importance: Bool) {
```
Comment (vtoloconnicov) [inline ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:42]: if you have 3+ parameters, create a struct and pass it here

Comment (elanghel) [reply ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:42]: this mean that I have to creat a model? I understood correctly?

Comment (vtoloconnicov) [reply ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:42]: you can call it a model. but it will be as a struct at the top of the file

Comment (elanghel) [reply ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:42]: okay

Comment (vtoloconnicov) [reply ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:42]: you already have an Item model. use that

File: ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift [hunk 3]
```diff
@@ -42,34 +33,37 @@ class HomeTableViewCell: UITableViewCell {
...
+        isChecked = !isChecked
+        checkImage.tintColor = isChecked ? UIColor(named: "Check Color") : UIColor(named: "Background Color")
+        taskView.backgroundColor = isChecked ? UIColor(named: "Selected Task Color") : UIColor(named: "Task Color")
     }
     
     
-    func setup(title: String, importance: Bool) {
+    func setup(data: Item) {
+        
+        let title = data.title
```
Comment (vtoloconnicov) [inline ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:45]: if you use data.title in same method use as it is. if you want to store it globally and use elsewhere, don't put in local variable.

File: ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift [hunk 4]
```diff
@@ -42,34 +33,37 @@ class HomeTableViewCell: UITableViewCell {
...
+        let importance = data.isImportant
```
Comment (vtoloconnicov) [inline ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:46]: same

File: ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift [hunk 5]
```diff
@@ -42,34 +33,37 @@ class HomeTableViewCell: UITableViewCell {
...
+        let category = data.category
```
Comment (vtoloconnicov) [inline ToDoList/Modules/HomeScreen/Views/HomeTableViewCell.swift:47]: for what you need taskCategory then?

**Review by vtoloconnicov — APPROVED** [review]

**Pull request merged**